docs/cli-reference.md

```diff
@@ -3,19 +3,14 @@ slug: cli-reference
 append_help_link: true
 description: "Reference for the Semgrep command-line interface including options and exit code behavior."
 tags:
-    - Semgrep CLI
-    - Semgrep Code
-    - Team & Enterprise Tier
+    - Semgrep AppSec Platform
```

Comment:
I feel like this is a bit mistagged and that Semgrep CLI would be a better fit

Reply:
I removed the Semgrep CLI because the CLI is an interface, not a product (it used to be a "product-like", like Semgrep in CI)... this is a hard one.  Let me think.